### PR TITLE
Create navigation to app button

### PR DESCRIPTION
**Pull request by Yukit02**
**implementation details**

- Subject says it All
- Define App url in helper file

### CODE REVIEW

File: src/helpers/snsNavigation.ts [hunk 1]
```diff
@@ -13,10 +13,12 @@ export default function snsNavigation() {
 
     const goToTwitter = () => window.open('https://twitter.com/Clitch10', '_blank');
     const goToDiscord = () => window.open('https://discord.gg/pEaBPyydE4', '_blank');
+    const goToClitchApp = () => window.open('https://app.clitch.life', '_blank')
```
Comment (hoonsubin) [inline src/helpers/snsNavigation.ts:16]: Please use an anchor tag for opening external pages. Adding a function is not scalable.

```html
<a href="https://app.clitch.life" target="_blank" rel="noopener noreferrer"><button>[call to action button element]</button></a>
```

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/helpers/snsNavigation.ts (hunk 1 above)
Comment (Yukit02) [reply src/helpers/snsNavigation.ts]: I fixed it.
That is OK?

Comment (Yukit02) [reply src/helpers/snsNavigation.ts]: I'll change button in CommunitySectionComponent when this modificate ok

**Comment by hoonsubin** [comment]
I will change the entire development framework. This PR will be merged.

**Pull request merged**